### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yuriks627**
updated Getting Started

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,9 +90,10 @@ ganache-cli --mnemonic 'candy maple cake sugar pudding cream honey rich smooth c
 deploy contracts.
 
 ```sh
-npm i lerna -g
-npm i yarn -g
+sudo npm i lerna -g
```
Comment (syuhei176) [inline README.md]: `sudo` may be depend on the environments(ex Mac, Linux, Windows)
@tkmct How do you think?

Comment (tkmct) [reply README.md]: Yes it does!
In document, it’s no need to put sudo.

Comment (yuriks627) [reply README.md]: Noted! sorry ab that

Comment (tkmct) [reply README.md]: And also, you can install multiple package in one line.
You can do this.
npm install -g yarn lerna

**Comment by yuriks627** [comment]
DONE!! plz approve!

**Comment by syuhei176** [comment]
LGTM!

**Pull request merged**